types/vimeo__player/index.d.ts

```diff
@@ -43,13 +44,17 @@ export class Player {
     addCuePoint(time: number, data: VimeoCuePointData): VimeoPromise<string, UnsupportedError | RangeError | Error>;
     removeCuePoint(id: string): VimeoPromise<string, UnsupportedError | InvalidCuePoint | Error>;
     getCuePoints(): VimeoPromise<VimeoCuePoint[], UnsupportedError | Error>;
+    getBuffered(): VimeoPromise<number, Error>;
     getCurrentTime(): VimeoPromise<number, Error>;
     setCurrentTime(seconds: number): VimeoPromise<number, RangeError | Error>;
     getDuration(): VimeoPromise<number, Error>;
     getEnded(): VimeoPromise<boolean, Error>;
     getLoop(): VimeoPromise<boolean, Error>;
     setLoop(loop: boolean): VimeoPromise<boolean, Error>;
     getPaused(): VimeoPromise<boolean, Error>;
+    getPlayed(): VimeoPromise<boolean, Error>;
+    getSeekable(): VimeoPromise<boolean, Error>;
```

Comment:
[`getSeekable` returns an array of video time ranges that are seekable](https://github.com/vimeo/player.js/#getseekable-promisearray-error), should probably be `[number, number][]` because it returns an array of tuples.